src/admin/navigations/navigations.const.ts

```diff
@@ -0,0 +1,9 @@
+import { ROUTE_PARTS } from '../../shared/constants';
+
+export const NAVIGATIONS_PATH = {
+	NAVIGATIONS_OVERVIEW: `/${ROUTE_PARTS.admin}/${ROUTE_PARTS.menu}`,
```

Comment:
Maybe refactor `ROUTE_PARTS.menu` to `ROUTE_PARTS.navigation`

Reply:
fixed in commit on pr higher on the stack:
https://github.com/viaacode/avo2-client/pull/1683/commits/1084917d5c4b842be4ff0d47fac51a268eafa026